src/contents/ui/ScrollingText.qml

```diff
@@ -84,6 +88,9 @@ Item {
             }
             onToChanged: () => reset()
             onDurationChanged: () =>  reset()
+            onPausedChanged: (paused) => {
+                if (paused && (scrollResetOnPause || !scrollingEnabled)) reset()
```

Comment:
```suggestion
                if (paused && scrollResetOnPause) reset()
```

I think that this `scrollingEnabled` check is no longer needed if you apply the suggestion in the other comment.

Reply:
Done, also fixed a infinite loop caused by calling `reset()`, changed it to set `label.x = 0`

```qml
if (paused && scrollResetOnPause) label.x = 0
```